src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -76,7 +76,7 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
     onCloseHandler();
   };
 
-  const onChangeSubsetWeight = (value: number) => {
+  const onChangeSubsetWeight = (value: any) => {
```

Comment:
why any?

Reply:
Because the value that comes from the textfield change it's "unknown". If we use number, decimal point doesn't appear when the value is "2. "